utils/check_support_level.js

```diff
@@ -34,29 +38,32 @@ const getNextLink = (linkHeader) => {
 };
 
 const findSupportLevel = async (url) => {
-  let packAddition = false;
+  let newPack = false;
 
   const files = await fetchFilesFromGH(url);
-  const packNames = files.reduce((acc, file) => {
+
+  const configFiles = findMainPackConfigFiles();
+  files.forEach((file) => {
     if (file.filename.includes('config.yml') && file.status === 'added') {
-      packAddition = true;
-    }
-    if (file.filename.includes('packs/')) {
-      acc.push(file.filename.replace('packs/', '').split('/')[0]);
+      newPack = true;
     }
-    return acc;
-  }, []);
+  });
+
+  const configFilesToRead = configFiles
+    .map((config) => {
+      const configRelativePath = removeRepoPathPrefix(config);
+      const regex = new RegExp(`^${path.dirname(configRelativePath)}.*`);
+      const fileMatch = files.find((file) => file.filename.match(regex));
```

Comment:
If we're trying to determine if a file is the same, could we just use [`path.basename`](https://nodejs.org/api/path.html#path_path_basename_path_ext)?

```suggestion
      const fileMatch = files.find((file) => file === path.basename(config));
```

Reply:
This is checking to see if a file exists within a quickstart. 
So if I had changes like:
```
packs/apache/dashboards/screenshot1.png - Added
```
and I have a list of pack directories:
```
packs/apache
packs/browser
...
```
This allows us to confirm that the new screen shot is part of the `apache` pack and to then read the `apache/config.yml` to check the support level of that pack.

Reply:
Ahh gotcha. Sorry that wasn't obvious to me as I was scanning through the code. Thanks!